_episodes/02-concepts.md

```diff
@@ -62,34 +77,63 @@ preferably machine accessible.
 > one shares with colleagues and collaborators. However, such 
 > information requires significant human resources to interpret and 
 > translate into code. An alternate approach is to encode the metadata
-> using structured markup (RDF, JSON, XML). Often such markup can be 
+> using structured markup (e.g., RDF, JSON, XML). Often such markup can be 
```

Comment:
If RDF, JSON, XML are first time referred here, we can provide links to each of them here.